crates/sui-open-rpc/src/examples.rs

```diff
@@ -868,7 +867,7 @@ impl RpcExampleProvider {
             .collect::<Vec<_>>();
         let page = CoinPage {
             data: coins,
-            next_cursor: Some(next),
+            next_cursor: Some("abcd".to_string()),
```

Comment:
Do the values end up appearing in the examples at all? If so it would be good to use something that's recognisably Base64?

Reply:
while it may be base64 encoded, the fact that it is is not part of the api contract as it really does need to be "opaque".